packages/Icon/src/svg/colored-caution.svg

```diff
@@ -0,0 +1,6 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24">
+  <g fill="none" fill-rule="evenodd" transform="translate(2 2)">
+    <path fill="#E5A823" fill-rule="nonzero" d="M9.99499476,17.2611115 C9.30534008,17.2611115 8.74562035,16.6993014 8.74562035,16.0070712 C8.74562035,15.314841 9.30534008,14.753031 9.99499476,14.753031 C10.6846494,14.753031 11.2443692,15.314841 11.2443692,16.0070712 C11.2443692,16.6993014 10.6846494,17.2611115 9.99499476,17.2611115 Z M8.99549523,7.9812137 L10.9944943,7.9812137 L10.9944943,14.0006069 L8.99549523,14.0006069 L8.99549523,7.9812137 Z M19.8850432,18.568323 L10.8895469,0.510143569 C10.5507165,-0.170047856 9.43927297,-0.170047856 9.10044263,0.510143569 L0.104946326,18.568323 C-0.0489766111,18.879325 -0.0329846179,19.2485144 0.148924307,19.5444679 C0.331832732,19.8394182 0.652672101,20.02 0.999498459,20.02 L18.990491,20.02 C19.3373174,20.02 19.6581568,19.8394182 19.8410652,19.5444679 C20.0229741,19.2485144 20.0389661,18.879325 19.8850432,18.568323 Z"/>
```

Comment:
Any interest in making this `fill="fillColor"` so that it can be controlled by the `color` prop? Also, I don't see a white shape for the exclamation mark, but what if the icon is not on a white background? I imagine the `!` should still be white?

Reply:
This icon will be only used with 
![image](https://user-images.githubusercontent.com/38733362/81771597-4b732a80-9498-11ea-9ac9-3f13081d5bd7.png) yellow background, the border is for solving contrast issue, so I think freeze the border color and fill color is better than making it flexible. We don't have any ugly issues like a red fill color with a dark yellow border. BTW red fill color usually on a white background, so they can use the normal `Caution` icon.
For the exclamation mark, the current version is transparent too. I'll double check with designers, thank you!

Reply:
Confirmed with Fernanda, it's ok to leave it transparent since we only use this icon on light backgrounds and it's tiny